appyters/RNA_Level_Functional_Predictions/appyter.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://raw.githubusercontent.com/MaayanLab/appyter-catalog/master/schema/appyter-validator.json",
+  "name": "RNA_Level_Functional_Predictions",
+  "title": "RNA level functional predictions",
+  "version": "0.0.3",
+  "description": "",
+  "authors": [
+    {
+      "name": "Christine Yoon",
+      "email": "christineyoon@live.com"
+    }
+  ],
+  "url": "https://github.com/MaayanLab/appyter-catalog",
```

Comment:
Is there possibly a more relevant link? Perhaps your github repo if it's public? Otherwise this is okay.

Reply:
Hi Daniel, my bad about missing the descriptions above. About the URL, I do have a public repo but since I migrated (instead of copying) the notebook into here I have made a lot of changes, so that public repo is a bit outdated.

Reply:
It's fine then to leave it as this url.